{{cookiecutter.project_slug}}/README.md

```diff
@@ -34,29 +39,33 @@ this will build the entire project with all dependencies inside a docker contain
 
 We use `pytest` as test framework. To execute the tests, please run
 
-    python setup.py test
+    {% if cookiecutter.package_manager == 'poetry' %}poetry run pytest tests{% else %}python setup.py test{% endif %}
 
 To run the tests with coverage information, please use
 
-    python setup.py testcov
+    {% if cookiecutter.package_manager == 'poetry' %}poetry run pytest tests --doctest-modules --junitxml=junit/test-results.xml --cov=src --cov-report=xml --cov-report=html{% else %}python setup.py testcov{% endif %}
```

Comment:
Hmm, can we provide a shortcut for that command? Can you define sub-commands in poetry, like we do in `setup.py`?

If not, then I would at least shorten the command to

    poetry run pytest tests --cov=src --cov-report=xml --cov-report=html

Reply:
Unfortunately no :( I was very sad when I discovered this. Happy to shorten the command.